apps/eth-wrapper/src/components/EthBalance.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
+import { ethers } from 'ethers';
+import { useSafeAppsSDK } from '@gnosis.pm/safe-apps-react-sdk';
+import { Text } from '@gnosis.pm/safe-react-components';
+
+const EthBalance: React.FC = () => {
+    const [balance, setBalance] = useState("loading...");
+    const { sdk, safe } = useSafeAppsSDK();
+
+    async function fetchBalance() {
```

Comment:
I am not sure when useCallback is better, but we could probably also declare it inside the useEffect.

But currently if this fails the whole app crashes, right?

Reply:
I would have to write a test for that, as I have no idea. I think that `useEffect` doesn't play nicely with inlined `async` functions. That's why it's extracted. I don't know if there is a more appropriate pattern for this.